tests/test_predict.py

```diff
@@ -69,3 +89,22 @@ def test_predict(model_class, env_id, device):
 
         action, _ = model.predict(vec_env_obs, deterministic=False)
         assert action.shape[0] == vec_env_obs.shape[0]
+
+
+def test_dqn_epsilon_greedy():
+    env = IdentityEnv(2)
+    model = DQN("MlpPolicy", env)
+    model.exploration_rate = 1.0
```

Comment:
I assume this code would fail without the fixes (as per #586)? Also #586 author says they had to call `learn`, but I assume this is also covered here somehow?

Reply:
`learn` is not required, what is required is that it uses the random policy when calling `predict`.
And yes, it fails without the fix ;)